src/lib/get-topper-settings.js

```diff
@@ -98,12 +98,18 @@ const usePackageTopper = (content) => {
 	};
 };
 
-const useEditoriallySelectedTopper = (content) => {
+const useEditoriallySelectedTopper = (content, flags) => {
 	let backgroundColour;
+	let myFtButtonVariant = 'standard';
 
 	// Convert old palette colours to new palette colours from Methode
 	if (content.topper.layout === 'full-bleed-offset') {
 		backgroundColour = 'wheat';
+	} else if (content.topper.layout === 'deep-landscape') {
+		if (content.topper.backgroundColour === 'wheat') {
```

Comment:
If we want to for all light **topper.backgrounds** to be the property **background  = white** I think we should do this also for other light colors , like auto or pink. You can see below this code that we switch background to wheat .
Other possibility could be to force this behaviour on [origami template for deep-landscape](https://github.com/Financial-Times/origami/blob/c34c1b76513a3058d3b5c6890742802dea3c6894/components/o-topper/src/scss/themes/_deep-landscape.scss#L31) like we do with text shadow where $foreground variabe is transform into white or black depending on the background so we don't have to care about themes.

Reply:
Added this in